.github/workflows/publish-docker-hub.yaml

```diff
@@ -15,6 +15,5 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_ACCESS_TOKEN }}
           repository: unitytechnologies/datasetinsights
-          cache_froms: unitytechnologies/datasetinsights:latest
```

Comment:
Why don't we want to cache from the `latest` now?

Reply:
Docker build cache are [not used](https://github.com/Unity-Technologies/datasetinsights/runs/1054004170?check_suite_focus=true#step:4:70) when building docker images using Github action. I think this might cause similar error so it's better not to use cache for production images.